### PR TITLE
Fusion: add audio options to cosmetics

### PR DESCRIPTION
**Pull request by JeffGainsNGames**
Adds the following audio options:
image

### CODE REVIEW

**Comment by Miepee** [comment]
There should be some kind of graphical seperaet between the default stuff and the disabling stuff. Otherwise people might think that disabling is a default and it can be enabled back in-game

**Comment by JeffGainsNGames** [comment]
Would have a label for 
```
in-game:
(  ) mono (x) stereo
some-name-here:
[  ] disable x
[  ] disable y
```
would it be good to add another box around it, or is that too many boxes.

File: test/games/fusion/gui/test_fusion_cosmetic_patches_dialog.py [hunk 1]
```diff
@@ -41,3 +43,26 @@ def test_color_space(skip_qtbot):
...
+)
+def test_change_music_option(skip_qtbot, music_start_value: bool, option_to_click: str, music_end_value: bool) -> None:
+    cosmetic_patches = FusionCosmeticPatches(stereo_default=music_start_value)
+
+    dialog = FusionCosmeticPatchesDialog(None, cosmetic_patches)
+    skip_qtbot.addWidget(dialog)
+    str_to_option_map = {
+        "stereo_option": dialog.stereo_option,
+        "mono_option": dialog.mono_option,
+    }
```
Comment (Miepee) [inline test/games/fusion/gui/test_fusion_cosmetic_patches_dialog.py:63]: you should be able to use `getattr` for this. `getattr(dialog, option_to_click)`.

**Comment by Miepee** [comment]
> Would have a label for
> 
> ```
> in-game:
> (  ) mono (x) stereo
> some-name-here:
> [  ] disable x
> [  ] disable y
> ```
> 
> would it be good to add another box around it, or is that too many boxes.

How about:
```
Default Audio settings:
( ) mono (x) stereo

Audio volume:
- [ ] Mute SFX
- [ ] Mute Music
```
?

**Comment by JeffGainsNGames** [comment]
image

**Comment by Miepee** [comment]
Can you add a little bit of spacing between them?

**Comment by Miepee** [comment]
grafik

**Review by Miepee — APPROVED** [review]

**Pull request merged**